addon/components/acmidm-logout.hbs

```diff
@@ -1,5 +1,7 @@
+<button ...attributes {{on "click" this.logout}}>
```

Comment:
not sure if this component is really needed since it's just a button wrapper for ember-simple-auth's session invalidation logic?